qa/L0_jax_unittest/test.sh

```diff
@@ -13,6 +13,6 @@ pip install -r $TE_PATH/examples/jax/encoder/requirements.txt
 pytest -Wignore -v $TE_PATH/examples/jax/mnist
 
 # Make encoder tests to have run-to-run deterministic to have the stable CI results
-export XLA_FLAGS="--xla_gpu_deterministic_ops"
+export XLA_FLAGS="--xla_gpu_deterministic_ops ${XLA_FLAGS}"
```

Comment:
I would do:
```
export XLA_FLAGS="${XLA_FLAGS} --xla_gpu_deterministic_ops"
```
This way, if XLA_FLAGS change xla_gpu_deterministic_ops, you are sure to overwrite it.

Reply:
Nice suggestion! I updated it.